panel/src/components/Dialogs/UploadDialog.vue

```diff
@@ -125,6 +154,9 @@ export default {
 .k-upload-items {
 	display: grid;
 	gap: 0.25rem;
+	max-height: 64dvh;
```

Comment:
`dvh` isn't supported by all browsers yet, we probably need a `vh` fallback above.
Where does 64 come from? Any specific reason?